DigitalLearningSolutions.Web/Views/TrackingSystem/Centre/Administrator/Index.cshtml

```diff
@@ -27,19 +27,56 @@
   <div class="nhsuk-grid-column-three-quarters">
     <div class="nhsuk-grid-row">
       <div class="nhsuk-grid-column-full">
-        <h1 id="page-heading" class="nhsuk-heading-xl">Centre administrators</h1>
-        <div class="nhsuk-u-margin-top-8">
-          <vc:number-of-administrators centre-id="@Model.CentreId" />
-        </div>
+        <h1 class="nhsuk-heading-xl">Centre administrators</h1>
+
+        <vc:number-of-administrators centre-id="@Model.CentreId" />

```

Comment:
Having had a look at the screenshot, this Number of Administrators view component looks a little squashed in with the large amount of stuff for the searching/filtering etc. when it is collapsed. It might be worth either making the text for the expandable larger, or adding a little extra margin on the containing div to make its separation from the rest of the page a little more obvious.